frontend/src/metabase/query_builder/components/DimensionList.jsx

```diff
@@ -80,14 +80,15 @@ export default class DimensionList extends Component {
       onRemoveDimension,
     } = this.props;
 
-    const surpressSubDimensions =
+    // XXX: Likely a typo
+    const suppressSubDimensions =
```

Comment:
This is likely a typo since [#18800](https://github.com/metabase/metabase/pull/18800/files#diff-a747d1635f99a8170f6f7e9bfd95d5a2375e80241fc1c518131ee152805d1406R114)